app/boards/shields/zodiark/zodiark.zmk.yml

```diff
@@ -0,0 +1,14 @@
+file_format: "1"
+id: zodiark
+name: Zodiark
+type: shield
+url: https://splitlogic.xyz
```

Comment:
Is there a more direct link to this exact keyboard?

Reply:
I can direct to the Build Guide on my website, that should have a link to the PCB when the group buy goes live.